SetMultiScreensVideoMode.brs

```diff
@@ -0,0 +1,40 @@
+' Multi Screens VideoMode Plugin Example
+' Plugin function name - setvideomode
+' Plugin return type - As Array: see below example, it is plugin's responsibility to 
+' return the same format as GetScreenModes() returns. Then the presentation autorun will
+' pick up your ScreenConfigList and set video modes for your screens.
+
+Function setvideomode(videoModeInputs, bsp)
+
+	print "setvideomode_ForMultiScreens"
+  ' See details regarding multiScreen configuration - [MultiScreen VideoMode Configuration](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/1208025142/VideoMode+Multiscreen+Configuration)

```

Comment:
Minor - can you align the white space?
Also the [url](https://xxx) syntax only works for markdown file. Can directly use url in brightscript.